packages/core/src/lib/filter/index.ts

```diff
@@ -493,9 +503,10 @@ class Filter extends BaseProtocol implements IReceiver {
 }
 
 export function wakuFilter(
-  init: Partial<ProtocolCreateOptions> = {}
+  init: Partial<ProtocolCreateOptions> = {},
+  pubsubTopics: PubsubTopic[]
```

Comment:
isn't pubsubTopic part of `ProtocolCreateOptions` already?

Reply:
Yes but I think in this case `pubsubTopics` should be required, not optional.

Reply:
That way we can make `init` required with `pubsubTopics` in it

Reply:
> That way we can make `init` required with `pubsubTopics` in it

agreed

Reply:
updated in latest commit